solutions/standard/variables.tf

```diff
@@ -0,0 +1,86 @@
+# Common
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API Key"
+  sensitive   = true
+}
+
+# Namespace
+variable "use_existing_resource_group" {
+  type        = bool
+  description = "Indicates whether to use an existing resource group. If set to 'false', a new resource group will be created."
+  default     = false
+}
+
```

Comment:
your missing an optional prefix variable which should be used for all resource naming if passed